src/main/java/de/zalando/aruha/nakadi/config/JsonConfig.java

```diff
@@ -37,6 +39,8 @@ public ObjectMapper jacksonObjectMapper() {
         objectMapper.registerModule(enumModule());
         objectMapper.registerModule(new Jdk8Module());
         objectMapper.registerModule(new ProblemModule());
+        objectMapper.registerModule(new JodaModule());
+        objectMapper.configure(WRITE_DATES_AS_TIMESTAMPS , false);
```

Comment:
I think that if we are able to save it and then read it back into an object it's enough. Just test that this serialize-desserialize works 👍